incubator/group/keeper.go

```diff
@@ -35,10 +40,20 @@ const (
 	VoteByVoterIndexPrefix        byte = 0x42
 )
 
+type ProposalI interface {
+	orm.Persistent
```

Comment:
```suggestion
    codec.ProtoMarshaler
```

This would make `ProposalI` consistent with the rest of the SDK now that `ProtoMarshaler` is standardized there.